### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by CJ-Wright**
Search for comments in the meta.yaml

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@conda-forge/conda-smithy Ready for review!

File: conda_smithy/lint_recipe.py [hunk 1]
```diff
@@ -149,6 +149,14 @@ def lintify(meta, recipe_dir=None):
     except RuntimeError as e:
         lints.append(str(e))
 
+    # 13: There should be no comments
```
Comment (ocefpaf) [inline conda_smithy/lint_recipe.py]: We must allow, and in cases like patches even require, for some comments.

Comment (CJ-Wright) [inline conda_smithy/lint_recipe.py]: Ok is there a good way to limit that down?
I have gotten a fair amount of boilerplate requests from reviewers to remove comments from my recipes.

**Comment by jakirkham** [comment]
I think the challenge is people keep comments from the `example` recipe by accident. Thus reviewers spend time ask that this be removed as they are clearly remnants. Maybe there is a better way we could document this so that people understand to do this cleanup themselves? Alternatively a better tool of some form for generating compliant recipes would remove the need for comment cleanup. Just some options to consider.

**Comment by CJ-Wright** [comment]
I am going to close this and move any relevant changes over to conda-verify

**Pull request closed without merging**